### PR TITLE
Implemented: new message if user searches for a keyword that not matches with any result in completed, open and progress view

### LINKED ISSUE #138: When wrong order id is searched, a message of no orders present on the page appears
When we search an order with a wrong id or using an order ID which is not present on the open orders or in-progress page, the message that appears says "Central Warehouse doesn't have any outstanding orders right now." 

It should say something like that "The order is not present on this page. Please search on other pages."

### PR DESCRIPTION
**Pull request by sanskar345**
### Related Issues

Closes #138

### Short Description and Why It's Useful

"No results found for <keyword>. Try searching In Progress or Completed tab instead.
If you still can't find what you're looking for, try switching stores." message is displayed in the tab if the user search keyword does not match any result.

### Screenshots of Visual Changes before/after (If There Are Any)

message

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/ionic-bopis#contribution-guideline)

### CODE REVIEW

File: src/views/InProgress.vue [hunk 1]
```diff
@@ -155,7 +155,11 @@
           <ion-icon :icon="checkmarkDoneOutline" />
         </ion-fab-button>
       </ion-fab>
-      <div class="empty-state" v-else>{{ currentFacility.name }} {{ $t(" doesn't have any orders in progress right now.") }} </div>
+      <div class="empty-state" v-else>
+        <div v-if="searchedQuery == ''">{{ currentFacility.name }}{{ $t(" doesn't have any outstanding orders right now.") }}</div>
+        <div v-if="searchedQuery != ''">{{ $t( `No results found for ${searchedQuery}. Try searching In Open or Completed tab instead.` ) }}</div>
```
Comment (ymaheshwari1) [inline src/views/InProgress.vue:160]:
```suggestion
        <div v-if="searchedQuery != ''">{{ $t( `No results found for ${searchedQuery}. Try searching in Open or Completed tab instead.` ) }}</div>
```

File: src/views/Completed.vue [hunk 2]
```diff
@@ -123,7 +123,9 @@
         </ion-fab-button>
       </ion-fab>
       <div class="empty-state" v-else>
-        {{ currentFacility.name }}{{ $t(" doesn't have any completed orders right now.") }}
+        <div v-if="searchedQuery == ''">{{ currentFacility.name }}{{ $t(" doesn't have any outstanding orders right now.") }}</div>
+        <div v-if="searchedQuery != ''">{{ $t( `No results found for ${searchedQuery}. Try searching In Progress or Open tab instead.` ) }}</div>
```
Comment (ymaheshwari1) [inline src/views/Completed.vue:127]: Add internationalization.

File: src/views/Completed.vue [hunk 3]
```diff
@@ -123,7 +123,9 @@
...
+        <div v-if="searchedQuery != ''">{{ $t( `If you still can't find what you're looking for, try switching stores.` ) }}</div>
```
[the remark this answers is not in the document]
Comment (sanskar345) [reply src/views/Completed.vue:128]: done

File: src/views/Completed.vue (hunk 2 above)
Comment (sanskar345) [reply src/views/Completed.vue:127]: done

File: src/views/Completed.vue [hunk 4]
```diff
@@ -123,7 +123,8 @@
         </ion-fab-button>
       </ion-fab>
       <div class="empty-state" v-else>
-        {{ currentFacility.name }}{{ $t(" doesn't have any completed orders right now.") }}
+        <p v-if="searchedQuery == ''">{{ currentFacility.name }}{{ $t(" doesn't have any outstanding orders right now.") }}</p>
+        <p v-if="searchedQuery != ''">{{ $t( `No results found for`)}}{{ " " + searchedQuery}}{{ $t(`. Try searching In Progress or Open tab instead.`)}} <br> {{ $t( `If you still can't find what you're looking for, try switching stores.` ) }}</p>
```
Comment (ymaheshwari1) [inline src/views/Completed.vue:127]: Use a single string for the message.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/locales/en.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   " doesn't have any completed orders right now.": " doesn't have any completed orders right now.",
   " doesn't have any orders in progress right now.": " doesn't have any orders in progress right now.",
-  " doesn't have any outstanding orders right now.": " doesn't have any outstanding orders right now.",
+  " doesn't have any outstanding orders right now.": "{facilityName} doesn't have any outstanding orders right now.",
```
Comment (ymaheshwari1) [inline src/locales/en.json:4]:
```suggestion
  "doesn't have any outstanding orders right now.": "{facilityName} doesn't have any outstanding orders right now.",
```

File: src/locales/en.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   " doesn't have any completed orders right now.": " doesn't have any completed orders right now.",
   " doesn't have any orders in progress right now.": " doesn't have any orders in progress right now.",
```
Comment (ymaheshwari1) [inline src/locales/en.json:3]: Update the error handling for completed and in progress page as well.

File: src/views/OpenOrders.vue [hunk 7]
```diff
@@ -174,10 +174,14 @@ export default defineComponent({
...
   data () {
     return {
-      shipmentMethods: [] as Array<any>
+      shipmentMethods: [] as Array<any>,
+      searchedQuery: ''
     }
   },
   methods: {
+    getErrorMessage() {
+      return this.searchedQuery === '' ? this.$t(" doesn't have any outstanding orders right now.", { facilityName: this.currentFacility.name }) : this.$t( "No results found for . Try searching In Progress or Completed tab instead. If you still can't find what you're looking for, try switching stores.", { searchedQuery: this.searchedQuery, lineBreak: '<br />' })
```
Comment (ymaheshwari1) [inline src/views/OpenOrders.vue:183]:
```suggestion
      return this.searchedQuery === '' ? this.$t("doesn't have any outstanding orders right now.", { facilityName: this.currentFacility.name }) : this.$t( "No results found for . Try searching In Progress or Completed tab instead. If you still can't find what you're looking for, try switching stores.", { searchedQuery: this.searchedQuery, lineBreak: '<br />' })
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**